### PR TITLE
Restyling playcourse button

### PR DESCRIPTION
**Pull request by jerrydev91**
Hey! Code will look a bit nasty hah, but i was trying ti figure out why i cant apply borderRadius to the TouchableOpacity. But i guess LinearGradient component has issues with borderRadius..

### CODE REVIEW

File: components/BookThisCourseButton.js [hunk 1]
```diff
@@ -19,13 +19,18 @@ const styles = StyleSheet.create({
     position: 'absolute',
     bottom: 0,
     left: 0,
-    right: 0
+    right: 0,
+    top: 0,
```
Comment (peterpme) [inline components/BookThisCourseButton.js]: remove this one, only want it stuck to the bottom

File: components/BookThisCourseButton.js [hunk 2]
```diff
@@ -4,12 +4,12 @@ import { LinearGradient } from 'expo'
 
 const BookThisCourseButton = ({ onPress }) =>
   <TouchableOpacity style={styles.container} onPress={onPress}>
```
Comment (peterpme) [inline components/BookThisCourseButton.js]: add `overflow: hidden` to the container

File: components/BookThisCourseButton.js [hunk 3]
```diff
@@ -4,12 +4,12 @@ import { LinearGradient } from 'expo'
...
-    <View>
+    <View style={{borderRadius: 15, borderWidth: 1 }}>
```
Comment (peterpme) [inline components/BookThisCourseButton.js]: remove these styles

File: components/BookThisCourseButton.js [hunk 4]
```diff
@@ -4,12 +4,12 @@ import { LinearGradient } from 'expo'
...
       <LinearGradient
-        colors={['#216F3F', '#144E55']}
-        style={{ padding: 15, alignItems: 'center' }}
+        colors={['blue', 'blue']}
+        style={{ padding: 15, alignItems: 'center', borderRadius: 20, borderWidth: 1 }}
```
Comment (peterpme) [inline components/BookThisCourseButton.js]: remove these border styles

**Comment by jerrydev91** [comment]
Alright fixing those right away! :)

File: components/CourseName.js [hunk 5]
```diff
@@ -23,9 +23,9 @@ type Name = {
   style?: StyleObj
 }
 
-const CourseName = ({ name, style }: Name) =>
+const CourseName = ({ name, style, textStyle }: Name) =>
   <View style={[styles.container, style]}>
-    <Text style={styles.text}>
+    <Text style={styles.text, textStyle}>
```
Comment (peterpme) [inline components/CourseName.js]: `<Text style={[styles.text, textStyle]}>`

File: components/LocationArrowIcon.js [hunk 6]
```diff
@@ -1,7 +1,13 @@
...
 import { FontAwesome } from '@expo/vector-icons'
 
-const LocationArrowIcon = () =>
-  <FontAwesome name='location-arrow' size={12} color='black' />
+type Icon = {
+  color: string,
+}
+
+const LocationArrowIcon = ({ color }: Icon) =>
+  <FontAwesome style={{ backgroundColor: 'transparent' }} name='location-arrow' size={12} color={color} />
```
Comment (peterpme) [inline components/LocationArrowIcon.js]: `<FontAwesome backgroundColor='transparent'`

File: components/CourseDistance.js [hunk 7]
```diff
@@ -21,13 +21,15 @@ const styles = StyleSheet.create({
...
+  arrowColor: string
 }
 
-const CourseDistance = ({ distance, style }: Distance) =>
+const CourseDistance = ({ distance, style, textStyle, arrowColor }: Distance) =>
   <View style={[styles.container, style]}>
-    <LocationArrowIcon />
-    <Text style={styles.text}>
+    <LocationArrowIcon color={arrowColor} />
+    <Text style={styles.text, textStyle}>
```
Comment (peterpme) [inline components/CourseDistance.js]: `<Text style={[styles.text, textStyle]}>`

**Pull request merged**